CONTRIBUTING.md

```diff
@@ -186,7 +186,7 @@ In general, all pull requests must:
     the project (for example refactoring for modularisation);
   - Be well peer reviewed;
   - Have unit tests and functional tests where appropriate;
-  - Follow code style guidelines;
+  - Follow code style guidelines ([C++](doc/developer-notes.md), [function tests](test/functional/README.md));
```

Comment:
Nit: functional tests not function tests

Reply:
Fixed typo.